am-lib/src/test/java/integration/uk/gov/hmcts/reform/amlib/FilterResourceIntegrationTest.java

```diff
@@ -278,18 +286,19 @@ void whenExplicitAccessWithDifferentRelationshipSameAttributeAndDifferentPermiss
                     JsonPointer.valueOf(PARENT_ATTRIBUTE), ImmutableSet.of(CREATE, READ)))
                 .accessType(EXPLICIT)
                 .build())
-            .relationships(ImmutableSet.of(OTHER_ROLE_NAME, ROLE_NAME))
+            .relationships(ImmutableSet.of(otherRoleName, roleName))
```

Comment:
we have a relationship variable - either remove or use

Reply:
same comment as above